src/bin/server.rs

```diff
@@ -38,7 +38,10 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                     .map(Cow::Owned)
                     .expect("SENTRY_ENV_API must be set when using SENTRY_DSN_API"),
             );
-            opts.release = option_env!("CRATES_IO_GIT_COMMIT").map(Into::into);
+
+            if let Ok(commit) = dotenv::var("HEROKU_SLUG_COMMIT") {
+                opts.release = Some(commit.into());
+            }
```

Comment:
One final nit, I think this can be reduced back to it's original form with the new env var: `opts.release = option_env!("HEROKU_SLUG_COMMIT").map(Into::into);`

Reply:
yep, that seems to work. thanks :)